spec/validator.ml

```diff
@@ -987,20 +988,48 @@ struct
                    drop the signature request. *)
                 (None, [])
               else
-                let state', action, actor =
+                let state', actions' =
                   match signature.status with
+                  | Waiting_for_sign_request _ ->
+                      ( Waiting_for_sign_request { responsible = None },
+                        `Coordinator_sign (signature.epoch.group.id, message)
+                        :: actions )
+                  | Collecting_sign_nonces { id; nonces } ->
```

Comment:
Note: This case is currently different in the validator service. If this times out then the validator triggers a new signature request with a smaller set signers.

So this is something we should adjust in the Validator at some point

Reply:
This behaviour didn't change in this PR (it is [like this already](https://github.com/safe-research/shieldnet/blob/3425d11b88f97b51a1cc129e812d0d2919f62aaa/spec/validator.ml#L1018-L1025)). I am personally a fan of this, as signing moves forward more quickly in case nodes are down.